app-info/eos-extra.xml

```diff
@@ -1,20 +1,5 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version='1.0' encoding='utf-8'?>
 <components version="0.8" origin="eos-extra">
-  <component type="desktop" merge="append">
```

Comment:
Oh, this shouldn't be removed, but it's because the process changed since #29 was created.

Reply:
Weird, on my local copy that line is still there

Reply:
I should have put this a little lower. There are lots of these `<component type="desktop" merge="append">` elements. This one in particular was the `app.drey.Warp` app. I added a commit to put the metainfo file in place so that the next time the `eos-extra.xml` catalog is generated it's there.